docs/img-src/architecture.mmd

```diff
@@ -0,0 +1,11 @@
+%% Ca( for flowchart below
```

Comment:
This diagram doesn't make much sense, what's the addition on top of the bigger diagram ?
I think its a bit confusing